### PR TITLE
🌐 Add language setting API

### PR DESCRIPTION
**Pull request by WeiJun0827**

### CODE REVIEW

File: src/server/api/routes/users/v2/locale.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+const { authenticateV2Login } = require('../../../middleware/auth');
+const { setPrivateCacheHeader } = require('../../../middleware/cache');
+const { handleRestfulError } = require('../../../middleware/error');
+const { walletUserCollection } = require('../../../../modules/firebase');
+
+const { DEFAULT_LOCALE, AVAILABLE_LOCALES } = require('../../../constant');
+
+const router = Router();
+
+router.get('/locale', authenticateV2Login, async (req, res, next) => {
```
Comment (WeiJun0827) [inline src/server/api/routes/users/v2/locale.js:11]: Not sure if we need this API though

File: src/server/api/constant/index.js [hunk 2]
```diff
@@ -48,6 +48,9 @@ const CIVIC_LIKER_QUANTITY_OPTIONS = [1];
 
 const VERIFICATION_EMAIL_RESEND_COOLDOWN_IN_MS = IS_TESTNET ? 60000 : 3600000;
 
+const DEFAULT_LOCALE = 'en';
+const AVAILABLE_LOCALES = ['en', 'zh-Hant'];
```
Comment (nwingt) [inline src/server/api/constant/index.js:52]: can this read from nuxt.config.js?

File: src/server/api/routes/users/v2/locale.js (hunk 1 above)
Comment (nwingt) [reply src/server/api/routes/users/v2/locale.js:11]: Keep it, maybe useful

**Review by nwingt — COMMENTED** [review]
I think we need to set locale when login too

File: src/store/modules/wallet.js [hunk 3]
```diff
@@ -404,11 +404,12 @@ const actions = {
       commit(WALLET_SET_LIKE_BALANCE_FETCH_PROMISE, undefined);
     }
   },
-  async walletFetchSessionUserInfo({ commit }, address) {
+  async walletFetchSessionUserInfo({ commit, dispatch }, address) {
     try {
       commit(WALLET_SET_FOLLOWEES_FETCHING_STATE, true);
       const userInfo = await this.$api.$get(getUserV2Self());
       commit(WALLET_SET_USER_INFO, userInfo || { user: address });
+      dispatch('setLocale', userInfo.locale);
```
Comment (williamchong) [inline src/store/modules/wallet.js:412]: Not sure if we need to check if `userInfo.locale` exists and is a valid locale in `setLocale`

**Review by williamchong — APPROVED** [review]
I was imagining the check to be inside serLocale but this would work too

**Review by williamchong — APPROVED** [review]

**Pull request merged**